cms/envs/common.py

```diff
@@ -2294,3 +2294,18 @@
     # The variable represents the window within which a verification is considered to be "expiring soon."
     "EXPIRING_SOON_WINDOW": 28,
 }
+
+################# Settings for brand logos. #################
+LOGO_TRADEMARK_URL = None
+LOGO_TRADEMARK_URL_PNG = None
+LOGO_TRADEMARK_URL_SVG = None
+
+LOGO_URL = None
+LOGO_URL_PNG = None
+LOGO_URL_SVG = None
+
+LOGO_WHITE_URL = None
+LOGO_WHITE_URL_PNG = None
+LOGO_WHITE_URL_SVG = None
```

Comment:
It seems a bit weird to me that we're specifying template level assets (some that seem very site-specific, like `LOGO_WHITE_URL`) in common config that every Open edX instance is going to carry around.

A few questions:

1. Will other sites have `LOGO_TRADEMARK_URL` and the like? Or are these specific to our new theme?
2. Could it be specified directly in the template for our theme?
3. If it does need to be in settings, might it make more sense to specify it in some way that's more freeform (e.g. `LOGOS` as a dict and let people stuff any number of variants in it)?

Reply:
These are really good questions!

1. As we have changed the branding handlers as well which use these variables if other sites would like to override their logo URLs they can use these variables. I do not think it's a new theme-specific but it is providing an ability to specify absolute URL to assets. 
2. We have defined these variables in our internal repo, to be used at multiple places, not sure if it could be a good call to use them in templates.
3. Good call, I would lean towards @nasthagiri with respect to this question. 

Nimisha if you could provide some insight to these questions, it would be great.

Reply:
@awaisdar001 The `LOGO_WHITE_URL` isn't needed in `edx-platform`, right? We could keep it out.

Regarding @ormsbee's questions:

1. I'm not sure about the distinction between the trademark version and non-trademark version. Perhaps the UX team can provide more info on why we need to have 2 different versions.
2. Yes, good point. I'm not up-to-par on the latest state of our configuration implementation in edx-platform. Is there a reason they need to be defined in the `common.py` module?
3. Hmm.. it would be good to understand the recommended design pattern for our config settings. Keeping them decoupled is what I thought our SRE team is preferring.

@awaisdar001 Overall, you want to remove any configuration settings that aren't actually used in edx-platform. Otherwise, they will add confusion if they are in the code but not actually used, no?

Reply:
Got it, thanks. I will remove it.